addon/templates/components/frost-button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! Template for the frost-button component }}
```

Comment:
I'm fine with keeping these comment blocks in the header of files - no real harm and there is some obvious benefit for certain users.